back-end/middlewares/jwt-validators.js

```diff
@@ -0,0 +1,17 @@
+const { getUserIdFromToken } = require('../helpers/jwt-helpers');
```

Comment:
@mcmoonstruck @AndreEPaul  Since validation checks for express validator and JWT may be reused in each route, I've added middlewares to be usable for these routes so the code doesn't need to be repeated for each.

Reply:
👍 thanks! I think Andre had headed down this path, but I was getting some errors when I was testing locally (likely more related to my code than the middleware Andre wrote), and I removed this as middleware while I was troubleshooting. This makes sense to add in.